service/src/main/java/com/codeforcommunity/processor/BlocksProcessorImpl.java

```diff
@@ -144,16 +144,18 @@ private GetAssignedBlocksResponse getAssignedBlocksWithStatus(BlockStatus status
             .fetchInto(AssignedBlock.class));
   }
 
+  /*
+   * Never use this in production. Uncomment and use locally, if needed. Don't commit the uncommented code.
+   * TODO: Create superuser privilege level for system management.
+   */
   @Override
   public void resetAllBlocks(JWTData jwtData) {
-    if (jwtData.getPrivilegeLevel() != PrivilegeLevel.ADMIN) {
-      throw new AdminOnlyRouteException();
-    }
+    //    throw new AdminOnlyRouteException();
```

Comment:
Why would we get rid of the admin check? It's better than nothing even if we can't have the super user in this PR

Reply:
This shouldn't be committed